truffle-config.js

```diff
@@ -16,6 +16,24 @@ module.exports = {
   //    port: 7545,
   //    network_id: "*"
   //  }
-  //}
+  //},
+
   //
+  db: {
+    // Truffle-DB is disabled by default. 
+    enabled: false,
+    host: "127.0.0.1",
```

Comment:
Why do we need `host` here?

Reply:
It is a `truffle db serve` option and I added it because I'm currently sshing into my dev machine and can set the host to '0.0.0.0' to browse to it over my LAN.

Reply:
Ah, I see. That makes sense then!